fatal error: absl/debugging/leak_check.h: No such file or directory

build in Ubunutu22.04 fail:

```
bazel build babylon
```

`
In file included from bazel-out/k8-fastbuild/bin/src/babylon/concurrent/_virtual_includes/thread_local/babylon/concurrent/thread_local.h:3,
                 from bazel-out/k8-fastbuild/bin/src/babylon/concurrent/_virtual_includes/counter/babylon/concurrent/counter.h:3,
                 from src/babylon/concurrent/counter.cpp:1:
bazel-out/k8-fastbuild/bin/src/babylon/_virtual_includes/sanitizer_helper/babylon/sanitizer_helper.h:3:10: fatal error: absl/debugging/leak_check.h: No such file or directory
    3 | #include <absl/debugging/leak_check.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
`

--- a/src/babylon/BUILD
+++ b/src/babylon/BUILD
@@ -174,7 +174,6 @@ cc_library(
   includes = ['//src'],
   strip_include_prefix = '//src',
   deps = [
-    '@com_google_absl//absl/debugging:leak_check',
     ':environment',
   ],
 )

--- a/src/babylon/sanitizer_helper.h
+++ b/src/babylon/sanitizer_helper.h
@@ -1,9 +1,9 @@
 #pragma once
 
-#include <absl/debugging/leak_check.h>
 #include "babylon/environment.h"
 
 // clang-format off
+#include BABYLON_EXTERNAL(absl/base/config.h) // ABSL_HAVE_ADDRESS_SANITIZER ABSL_HAVE_LEAK_SANITIZER
 #include BABYLON_EXTERNAL(absl/base/attributes.h) // ABSL_ATTRIBUTE_ALWAYS_INLINE
 // clang-format on
 

